envi/cli.py

```diff
@@ -476,6 +478,17 @@ def do_script(self, line):
         locals = self.getExpressionLocals()
         locals['argv'] = argv
 
+        if line.startswith("?"):
```

Comment:
Why do not include readline support to enable autocompletion and other helper functions?